index.html

```diff
@@ -0,0 +1,170 @@
+<body id="hideOverflow">
+    <div class="social">
+        <div class="social-media">
+            <i class="fa-brands fa-facebook-f"></i>
+            <i class="fa-brands fa-twitter"></i>
+        </div>
+        <ul class="top-nav">
+            <li><a href="#" class="top-nav-link">English</a></li>
+            <li><a href="#" class="top-nav-link">My Page</a></li>
+            <li><a href="#" class="top-nav-link">Logout</a></li>
+        </ul>
+    </div>
+    <header class="header" id="header">
+        <div class="logo">
+            <a href="index.html"><img class="logo-img" src="images/webtekhy-icon.png" alt="webtekhy icon"></a>
+        </div>
+        <nav id="menu_bar" class="navbars">
+            <i id="menuBarImg" class="fa fa-bars" aria-hidden="true"></i>
+            <div id="close_bar" class="close-menu">
+                <i id="close_menu" class="close" >&times;</i>  
+            </div>
+            <ul id="menu_list" class="navis">
+                <li class="nav-item"><a href="about.html" class="nav-link abt-nav">About</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">Program</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">Join</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">Sponsor</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">News</a></li>
+                <li class="nav-item"><a href="#" class="nav-link wt-nav">WT Campaign</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section class="title-section">
+            <div class="details">
+                <p class="quote-text">
+                "Hello! Sharing World"
+                </p>
+                <h1 class="title">
+                    WEBTEKHY UNIVERSAL BOOTCAMP 2022
+                </h1>
+            </div>
+            
+            <div class="descriptions">
+                <p class="description">
+                    A fun festival where people from more than 80 countries who believe in the value of sharing and openness and 
+                    create positive change gather and share their thoughts will be held in Korea in October.
+                </p>
+            </div>
+
+            <h2 class="date">
+                2022.05.16(MON) ~ 20(FRI)
+            </h2>
+            <p class="location">
+                @ National Theartre of Nigeria, Kwara Hotel and more
+            </p>
+        </section>
+
+        <section class="program-section">
+            <h2 class="program-title">Our Program</h2>
+            <div class="bottom-line"></div>
+            <div class="programs">
+                <article class="prog program-1">
+                    <div class="program-icon-img transparency">
+                        <img src="images/program_icon_01.png" alt="Lecture Icon">
+                    </div>
+                    <h3 class="prog-title title-1">Lecture</h3>
+                    <div class="about-prog">
```

Comment:
- [ ] [optional] I am marking this change as optional because it was missed by the previous reviewer but I strongly suggest considering them, When we hover the mouse over cards a white border around cards should appear as on the design, Please apply hover effects to your cards so the cards show a white border when we hover the mouse over them.

### See the image for reference 

![image](https://user-images.githubusercontent.com/83803414/157689981-91c089cf-d233-49a4-b0d5-3e74f7a6d17b.png)